doc/050_restore.rst

```diff
@@ -156,6 +156,31 @@ e.g.:
 
     $ restic -r /srv/restic-repo dump --path /production.sql latest production.sql | mysql
 
+This example assumes you attached an absolute path, which means it coincides with the
```

Comment:
The path stored in the snapshot metadata (==attached path) is always absolute. I think you mean `you ran a backup using an absolute path`

Reply:
This is clearer - I will change that.

Reply:
changed